playbooks/cluster/openshift/vars/gluster.yml

```diff
@@ -1,11 +1,10 @@
 # Namespace to create for the GlusterFS and Heketi resources
 openshift_storage_glusterfs_namespace: glusterfs
-# Do not create a StorageClass; a custom StorageClass will be created via APB
-# for use by kubevirt
-openshift_storage_glusterfs_storageclass: false
+# use opneshift glusterfs StorageClass
+openshift_storage_glusterfs_storageclass: true
+# set glusterfs as default StorageClass
+openshift_storage_glusterfs_storageclass_default: true
 # glusterblock functionality is not supported outside of Logging/Metrics
 openshift_storage_glusterfs_block_deploy: false
-# (Temp) Use a custom Heketi image to enable cloning support
-openshift_storage_glusterfs_heketi_image: gluster/heketiclone
 # Disable any other default StorageClass
-openshift_storageclass_default: false
+openshift_storageclass_default: false
```

Comment:
Add back the newline at the end of the file.

Reply:
done